### DISCUSSION TITLE
Third Person to First Person camera view point offset?

### DISCUSSION THREAD

**Discussion by xkamronx** [Sixze/ALS-Refactored#392] [Q&A]
Would it be possible to fix the view modes so they're both looking at the same position?

If you're in third person and look directly at a point, then swap to first person, it moves where you're looking to the left by a decent amount. This can also be worsened by adjusting the third person camera offset to a large amount.

**Comment by ameaninglessname** [comment]
Basically, you need to implement the `aim lock` functionality, which adjust the camera rotation every tick based on `current camera location` and `aim lock point`(look at rotation)

**Reply by xkamronx** [reply]
Just to clarify, this isn’t something already built in to ALS, but something that needs to be added directly correct?

**Reply by ameaninglessname** [reply]
Yes. We need to build it ourselves, there might be good alternatives on the fab.

(Sorry, I didn't saw this until now.)
